### ISSUE TITLE
Allow Screenly to be installed with a user other than 'pi'

### ISSUE THREAD

**Issue by vpetersson** [Screenly/screenly-ose#1621]
Previous to Rasbian OS/Bullseye, the installer had the user 'pi' hard coded into the system, and we made assumptions around this. Since this is no longer true, there are a number of ways the installer will break if it is installed using a different user.

Breakage has been reported in a few issues, including in #1619.

**Comment by MangNPB** [comment]
I tried the latest bash script install and 'pi' user is still required. is there a different installing .sh or method to install it on other user?

**Comment by vpetersson** [comment]
Yes, this ticket is still opened, so it isn't resolved.

**Comment by ealmonte32** [comment]
@MangNPB ,

This is a very big task because you would need to have part of the install script get the current username and then replace every single ansible task or code that mentions `/home/pi` and replace it with the variable that will obtain the name of the current user so that it would always be dynamic.

**Comment by vpetersson** [comment]
Yes, and not only that, we would also make sure all `sudoers` is configured properly etc.

**Comment by MangNPB** [comment]
Big thanks to all those who are in the works for making this change as it's one of the brute force first choice of username.
Any estimations of when will this be done?

**Comment by vpetersson** [comment]
It's not super highly prioritized given how easy it is to work around.

**Comment by vpetersson** [comment]
Resolved in https://github.com/Screenly/screenly-ose/pull/1662
